### DISCUSSION TITLE
Guidance Request on Implementing a View Count System

### DISCUSSION THREAD

**Discussion by guilhermeguidetti** [pocketbase/pocketbase#4170] [Q&A] [answered]
Hello!

Firstly, I understand this might not be the ideal platform for such inquiries as it's not a traditional issue, but I'm hoping to gain some insights from your expertise with PocketBase.

I am currently working on a project where I am using PocketBase as the backend service. My application features a feed similar to Twitter, and I am looking to implement a system to count the views of each post.

The initial approach I considered was to send a separate request to increment the view count in the database for each post loaded on the user's feed. However, I am concerned that this method might overload the backend, especially with a high volume of traffic.

I am reaching out to seek your advice or suggestions on the best way to implement this feature using PocketBase.

I would greatly appreciate any insights or recommendations you could provide. If there are existing features or best practices within PocketBase that could facilitate this implementation, please do let me know.

**Comment by nokia6290** [comment]
Not a pocketdb expert here. but From DB fundamentals. Just have an object(db table) to count these views somewhere OR have a query that counts them for u each time.

Its not really a pocketdb thing. Its a general db pattern

**Comment by guilhermeguidetti** [comment]
I have a column in PocketBase dedicated to storing the view count for each post. My challenge, however, lies in the need to send an individual request to update the view count for each "tweet", akin to updating 'tweets'. I'm uncertain if PocketBase allows for a single request to simultaneously update a group of "tweets". This is the crux of my inquiry – to find a more efficient method, if one exists within PocketBase, for handling this update process. Alternatively, do you think continuing with the current approach of individual updates is manageable and won't cause issues, even with increased traffic?

**Reply by ganigeorgiev** [reply, marked as answer]
You can explore some of the suggestions in https://github.com/pocketbase/pocketbase/discussions/4012.
